Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/navigation/NavDrawerActionListener.java

```diff
@@ -0,0 +1,7 @@
+package com.novoda.todoapp.navigation;
+
+public interface NavDrawerActionListener {
```

Comment:
Renamed in https://github.com/novoda/spikes/pull/138/commits/ed8393e2673e55d5df2079d36aa37e7375044de2